src/store/api.ts

```diff
@@ -418,35 +428,32 @@ export const createDeal = multiAsync(
 
 export const activateDeal = multiAsync(async (connection: Connection, wallet: Wallet) => {
 	const program = constructProgram(connection, wallet);
-	const _userUSDCTokenAccount = getUserUSDCTokenAccount(connection, wallet);
+	const _userAssociatedUSDCTokenAddressPK = getAssociatedUSDCTokenAddressPK(connection, wallet);
```

Comment:
This is not correct. We should be able to get the associated token address for an arbitrary borrower defined in the deal, not only the connected wallet.

Reply:
https://www.notion.so/credix/Allow-for-inputting-the-key-of-a-random-borrower-when-creating-a-deal-596483f76015436cadebbc242f46daa8 will be handled here